### DISCUSSION TITLE
Devices off line

### DISCUSSION THREAD

**Discussion by Marcus235** [milo526/homebridge-tuya-web#299] [Q&A]
Using Hoobs 4 and since today all my Smart Life devices appear off line. In the Smart Life app everything is on line and devices are registered in tuya plug in. 
Any comments or ideas? 
![DD52D364-B4BB-4B95-B4E0-A2EACC360186](https://user-images.githubusercontent.com/61418593/125593779-b2cd67c7-ee44-4456-ae06-4a95abc58526.jpeg)
![078FC274-D221-4819-8ACF-A383301E8EBC](https://user-images.githubusercontent.com/61418593/125593787-822e419c-5507-469e-9c13-bfc77fc1cd5c.jpeg)

**Comment by mbierman** [comment]
Do these devices work in the Smart Life app?

**Reply by Marcus235** [reply]
Yes. In the Smart Life app everything works.

**Reply by mbierman** [reply]
Just to confirm the smartlife app you are using matches the Platform setting in Homebridge/Hoobs? the accounts on different platforms are not interchangeable. 

```
            "platform": "TuyaWebPlatform",
            "name": "Smart Life",
            "options": {
                "username": "***",
                "password": "***",
                "countryCode": "1",
                "platform": "smart_life",
                "pollingInterval": 600
            },
            "scenes": false
        },
```

**Reply by Marcus235** [reply]
Yes. And everything worked like a charm before. Up until now.

**Reply by Marcus235** [reply]
It's seems the problem is resolved after removing power from the device. Not sure if this is temporary
